lib/screens/AddGroupScreen.dart

```diff
@@ -156,45 +155,49 @@ import 'SignInScreen.dart';
     }
   }
 
+class FriendRow extends StatelessWidget {
+  final ProfileManager profileMgr;
+  final friendIdx;
+  final AddGroupScreenState state;
```

Comment:
I think we shouldn't pass state like this, instead pass the data we need. If we want to use the parent state for some reason, I think using a callback is better. 

This SO answer is clear: https://stackoverflow.com/a/50085960

Reply:
Done